### PR TITLE
[netatmo] eventBuffer in SecurityCapability should not be static

### PR DESCRIPTION
**Pull request by lolodomo**
Also fix few findings by code analysis

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/SecurityCapability.java [hunk 1]
```diff
@@ -47,7 +47,7 @@
 class SecurityCapability extends RestCapability<SecurityApi> {
     private final Logger logger = LoggerFactory.getLogger(SecurityCapability.class);
 
-    private static final Map<String, HomeEvent> eventBuffer = new HashMap<>();
+    private static final Map<String, HomeEvent> EVENT_BUFFER = new HashMap<>();
```
Comment (lolodomo) [inline bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/SecurityCapability.java:50]: @clinique : I have a BIG doubt if it must be static or not ?

Comment (clinique) [reply bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/SecurityCapability.java:50]: No, it should not be static.

Comment (lolodomo) [reply bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/SecurityCapability.java]: Ok, fixed.

**Review by jlaur — APPROVED** [review]
Thanks! Would it be possible to assess what could have been the consequence of this bug? Then we might be able to find possible connected issues and modify PR title to be user-oriented.

**Comment by clinique** [comment]
We could imagine a user having multiple houses, in this case, the event buffer would have mixed events coming from both houses, leading to a potential loss of events.

**Comment by jlaur** [comment]
@lolodomo - can you either remove "Bug" label for being excluded from the release notes, or adapt the title according to the possible user impact described by @clinique?

**Comment by lolodomo** [comment]
Title updated

**Pull request merged**